src/labels/CustomLabels.labels

```diff
@@ -2391,6 +2391,14 @@
         <shortDescription>Contains message indicating one installment has been selected to skip</shortDescription>
         <value>You&apos;ve selected 1 installment.</value>
     </labels>
+    <labels>
+        <fullName>RD2_PauseStatusMustExist</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when Recurring Donation Status Pause picklist value does not exist</shortDescription>
+        <value>NPSP is unable to update Status and Status Reason on paused Recurring Donations. You must add the picklist value &apos;Pause&apos; to the Status field.</value>
+    </labels>
```

Comment:
@sam-knox Can you review the new custom label

Reply:
@andrewyu-salesforce - I realize this is a little late in the game, but can we use "Paused" instead of "Pause" for the new picklist value? That way it matches the tense of the other values.

@mpusto